src/agent/src/cdh.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+// Confidential Data Hub client wrapper.
+// Confidential Data Hub is a service running inside guest to provide resource related APIs.
+// https://github.com/confidential-containers/guest-components/tree/main/confidential-data-hub
+
+use anyhow::Result;
+use derivative::Derivative;
+use protocols::{
+    sealed_secret, sealed_secret_ttrpc_async, sealed_secret_ttrpc_async::SealedSecretServiceClient,
+};
+const CDH_ADDR: &str = "unix:///run/confidential-containers/cdh.sock";
```

Comment:
!!!NIT ALERT!!! I would call it `CDH_SOCKET` as ADDR can be misleading (think the cdh's http address).

BTW, `src/main.rs` already defines `CDH_SOCKET`. I think it would be good to refactor the main's code to re-use this `CDH_SOCKET`. You may want to actually split in those variables, `CDH_SOCKET` and `CDH_SOCKET_URI` just like:

```
const AA_ATTESTATION_SOCKET: &str =
    "/run/confidential-containers/attestation-agent/attestation-agent.sock";
const AA_ATTESTATION_URI: &str = concatcp!(UNIX_SOCKET_PREFIX, AA_ATTESTATION_SOCKET);
```